### PR TITLE
🌱 adding jmrodri as reviewer

### PR DESCRIPTION
**Pull request by jmrodri**
**Description**

Adding myself to the reviewer's list

**Motivation**
Get the permissions

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -16,6 +16,7 @@ aliases:
   # approvers & admins -- those count too via the OWNERS file)
   kubebuilder-reviewers:
     - joelanford
+    - jmrodri
```
Comment (camilamacedo86) [inline OWNERS_ALIASES:19]:
```suggestion
    - jmrodri
    - joelanford
```

**Review by camilamacedo86 — APPROVED** [review]
/approved 
/lgtm 

when pass in the ci check

**Comment by jmrodri** [comment]
@camilamacedo86 I submitted for access in this issue: https://github.com/kubernetes/org/issues/3399#issuecomment-1118746284 and this PR was created and shows me being added to the kubernetes-sigs org https://github.com/kubernetes/org/pull/3405 

https://github.com/palnabarun/org/blob/5b66d9710f73e96aa4b296d20aed9c463b4775d3/config/kubernetes-sigs/org.yaml#L347

**Pull request merged**